backend/import_status/get.py

```diff
@@ -41,27 +47,25 @@ def get_import_status(event: JsonObject) -> JsonObject:
     assert "status" in step_function_resp, step_function_resp
     LOGGER.debug(json.dumps({"step function response": step_function_resp}, default=str))
 
-    upload_response: JsonObject = {"status": "Pending", "errors": []}
-
-    # only check status of upload if step function has completed
-    if step_function_resp["status"] == "SUCCEEDED":
-        assert "output" in step_function_resp, step_function_resp
-        step_functions_output = json.loads(step_function_resp["output"])
-
-        assert (
-            "s3_batch_copy" in step_functions_output
-            and "job_id" in step_functions_output["s3_batch_copy"]
-        ), step_function_resp
+    step_func_input = json.loads(step_function_resp["input"])
+    step_functions_output = json.loads(step_function_resp.get("output", "{}"))
```

Comment:
We should probably use the same prefix for both variables, either `step_func` or `step_function`.